NEW_MIPS_PROCESS.md

```diff
@@ -0,0 +1,329 @@
+# On-Chain Voting Results for [MIPs](https://github.com/MinaProtocol/MIPs)
+
+This article details the calculation of Mina’s on-chain voting results.
+
+Equipped with the tools and knowledge, anyone can independently verify the code, logic, and
+results. We always strive for correctness in our code and exhaustively test.
+
+All the code for this project is open source ❤️ and available on [GitHub](https://github.com/o1-labs/mina-on-chain-voting).
+
+Find an issue or a bug? Have a question or suggestion? We’d love to get your [feedback](https://github.com/o1-labs/mina-on-chain-voting/issues)!
+
+# Table of Contents
+- [API Reference](#api-reference)
+- [Calculation of Mina’s On-Chain Voting Results](#calculation-of-minas-on-chain-voting-results)
+  - [Overview](#overview)
+  - [Results Calculation Instructions](#results-calculation-instructions)
+  - [Calculation steps](#calculation-steps)
+    - [Obtain staking ledger](#obtain-staking-ledger)
+    - [Aggregate voting stake](#aggregate-voting-stake)
+    - [Obtain and parse votes](#obtain-and-parse-votes)
+    - [Calculate weighted voting results](#calculate-weighted-voting-results)
+    - [Adjust Votes and Voting Stake with Non-Delegating Voters](#adjust-votes-and-voting-stake-with-non-delegating-voters)
+  - [Conclusion](#conclusion)
+
+## How to add a new MIP
+Adding a MIP in the OCV will ensure that the MIP's progress is visible and that the votes are counted and weighed correctly.
+To add a new MIP, please follow the following steps:
+1. Go to the [MIP list file in the OCV repo](https://github.com/o1-labs/mina-on-chain-voting/blob/main/server/proposals/proposals.json)
+2. Click on the pencil icon to edit the file
+3. Add a new entry to the list of proposals, following the format of the existing entries
+   - the `id` should be the next available integer
+   - the `title` should be a descriptive title of the MIP
+   - the `key` should be the MIP number (e.g. `MIP5`) - This will be the _memo_ used for voting this MIP
+   - the `start_time` and `end_time` should be the timestamps when voting starts and ends for this MIP
+   - the `url` should be a link to the MIP's proposal on the [Mina Protocol MIPs repository](https://github.com/MinaProtocol)
+   - the network should be `mainnet` or `devnet`
+   - the `ledger_hash` must be the ledger hash of the epoch after the voting epoch (e.g. for voting in epoch 53, use the ledger hash of epoch 55)
+4. Once you have added the new entry, scroll down to the bottom of the page and click on the "Propose changes" button
+5. In the next page, add a descriptive title and description for your changes, then click on the "Create pull request" button
+6. Once the pull request is reviewed and approved, it will be merged into the main branch and the new MIP will be added to the [OCV dashboard](https://ocv.minaprotocol.com/)
```

Comment:
Is this automatic when it gets merged or do we need to deploy for the changes to take place?

Reply:
This is automatic.